heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/service/OAuthInterceptorService.java

```diff
@@ -470,7 +470,7 @@ private void validateInProvider(Provider provider, String clientId, String acces
                 ResponseEntity<String> entityResponse = template
                     .exchange(uriComponentsBuilder.build().encode().toUri(), HttpMethod.POST, createHttpEntity(uriComponentsBuilder, provider.getProviderParams()), String.class);
 
-                HeimdallException.checkThrow(!(Series.valueOf(entityResponse.getStatusCodeValue()) == Series.SUCCESSFUL), ExceptionMessage.PROVIDER_USER_UNAUTHORIZED);
+                HeimdallException.checkThrow((Series.valueOf(entityResponse.getStatusCodeValue()) != Series.SUCCESSFUL), ExceptionMessage.PROVIDER_USER_UNAUTHORIZED);
```

Comment:
Remove parenthesis of the first parameter

Reply:
Done